open_lm/model.py

```diff
@@ -202,9 +245,14 @@ def __init__(self, layer_id, args: Params):
         elif args.ffn_type == "gelu":
```

Comment:
Could we also support fp8 for swiglu above? We can make a copy of the Swiglu class in this file. Here's the source for Swiglu https://github.com/facebookresearch/xformers/blob/7f8c290183344343771f4e1d945a8ce10a9500ff/xformers/ops/swiglu_op.py#L430

Reply:
@rams16592 seems like the recursive replace linear patten should take care of this automatically. a function like this seems like it would be great and we can exclude certain linears that need to be higher precision for stability. this function has an include field instead of exclude, but hopefully that's easy to flip: 
https://github.com/mlfoundations/open_clip/blob/73fa7f03a33da53653f61841eb6d69aef161e521/src/open_clip/utils.py#L65

Reply:
Applied this change in the latest commit. Excluding the last output Linear layer from the conversion to TE Linear as its running into errors.

Reply:
Applied this change in the latest commit. Excluding the last output Linear layer from the conversion to TE Linear as its running into errors.